identity-wallet/src/state/extensions/actions.rs

```diff
@@ -0,0 +1,19 @@
+use crate::{reducer, state::{actions::ActionTrait, extensions::reducers::test_feat_state, reducers::Reducer}};
+use ts_rs::TS;
+
+
+/// Action to test the extension field.
+#[derive(serde::Serialize, serde::Deserialize, Debug, TS, Clone)]
+#[ts(export, export_to = "bindings/actions/Test.ts")]
```

Comment:
This creates the bindings in `identity-wallet/bindings/actions/Test.ts`, but since we won't be needing bindings for this struct this line can be omitted :)